tasks/build-react.js

```diff
@@ -3,19 +3,114 @@ Copyright 2022 Adobe. All rights reserved.
 This file is licensed to you under the Apache License, Version 2.0 (the "License");
 you may not use this file except in compliance with the License. You may obtain a copy
 of the License at http://www.apache.org/licenses/LICENSE-2.0
-
 Unless required by applicable law or agreed to in writing, software distributed under
 the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 OF ANY KIND, either express or implied. See the License for the specific language
 governing permissions and limitations under the License.
 */
 
+/**
+ * @fileoverview This task generates React wrapper components for Spectrum Web Components
+ * using the Custom Elements Manifest (CEM) analyzer.
+ *
+ * @description
+ * This script:
+ * 1. Removes any existing 'react' directory for clean generation
+ * 2. Analyzes all workspace packages (except ignored ones) using CEM
+ * 3. Generates React wrapper components using a custom configuration
+ * 4. Creates icon-specific React wrappers using a separate script
+ *
+ * @output
+ * - Info: "Removed react directory."
+ * - Info: "Generated React wrapper."
+ * - Error: "Error removing react directory:" followed by error details
+ * - Error: "Error generating React wrapper:" followed by error details
+ * - Error: "Error running command in [path]:" followed by error details
+ */
+
+import { execSync } from 'child_process';
+import fs from 'fs';
+import path from 'path';
+import { dirname } from 'path';
+import { fileURLToPath } from 'url';
+
 import fg from 'fast-glob';
+import { cli } from '@custom-elements-manifest/analyzer/cli.js';
 import { buildPackage } from './ts-tools.js';
+import { generateIconWrapper } from '../scripts/cem-plugin-react-wrapper.js';
```

Comment:
Similarly to the build-css script, this one didn't need these abstractions or to call the exec functionality when the node utilities exist.